Index: openwrt/feeds/luci/modules/luci-mod-status/htdocs/luci-static/resources/view/status/load.js
===================================================================
--- openwrt.orig/feeds/luci/modules/luci-mod-status/htdocs/luci-static/resources/view/status/load.js
+++ openwrt/feeds/luci/modules/luci-mod-status/htdocs/luci-static/resources/view/status/load.js
@@ -215,7 +215,7 @@ return view.extend({
 				throw new Error(response.statusText);
 
 			return E('div', {
-				'style': 'width:100%;height:300px;border:1px solid #000;background:#fff'
+				'style': 'width:100%;height:300px; solid #000;'
 			}, E(response.text()));
 		});
 	},
